llvm/lib/Transforms/Utils/ProfileVerify.cpp

```diff
@@ -26,6 +26,18 @@ using namespace llvm;
 static cl::opt<int64_t>
     DefaultFunctionEntryCount("profcheck-default-function-entry-count",
                               cl::init(1000));
+static cl::opt<bool>
+    AnnotateSelect("profcheck-annotate-select", cl::init(true),
+                   cl::desc("Also inject (if missing) and verify MD_prof for "
+                            "`select` instructions"));
+static cl::opt<uint32_t> SelectTrueWeight(
+    "profcheck-select-true-weight", cl::init(2U),
```

Comment:
nit: profcheck-default-select-true-weight to make it sound similar to the default function entry count one?